src/manage_nsfs/manage_nsfs_help_utils.js

```diff
@@ -17,200 +17,181 @@ Usage:
+Flags:
+--name <string or number>                                                                   The name of the account
+--access_key <string>                                     (optional)                        The access key of the account (identify the account instead of name)
 `;
 
-const ACCOUNT_OPTIONS_STATUS = `
-Account
-account status [options...]:
-    print the account's details, such as name, email, creation date, UID, GID (or user details), etc.
-    
-
-    # required
-    --name <name>                                                                               The name of the account.
-                                                                                                (can identify the account by --access_key option)
+const ACCOUNT_FLAGS_STATUS = `
+Usage:
+account status [flags]
 
-    # optional
-    --show_secrets                                  (default false)                             Print the access key and secret key of the account.
+Flags:
+--name <string or number>                                                                   The name of the account
+--access_key <string>                                     (optional)                        The access key of the account (identify the account instead of name)
+--show_secrets                                            (optional)                        Print the access key and secret key of the account
 `;
 
-const ACCOUNT_OPTIONS_LIST = `
-Account
-account list [options...]:
-    list all accounts' names
-
-    # optional
-    --wide                                          (default false)                             Print the additional details for each account.
-    --show_secrets                                  (default false)                             Print the access key and secret key of each account (only when using option --wide).
-    
-    # optional filters
-    --uid                                                                                       Filters the list based on the provided UID.
-    --gid                                                                                       Filters the list based on the provided GID.
-    --user                                                                                      Filters the list based on the provided FS user distinguished name.
-    --name                                                                                      Filters the list based on the provided account name.
-    --access_key                                                                                Filters the list based on the provided account access_key.
+const ACCOUNT_FLAGS_LIST = `
+Usage:
+account list [flags]
+
+Flags:
+--wide                                                    (optional)                        Print the additional details for each account
```

Comment:
@shirady it will be good if can mention the default value for wide and show_secrets flags.

Reply:
@naveenpaul1 Would you please add the suggestion?
If I would change it to:
```suggestion
--wide                                                    (optional)                        Print the additional details for each account (default is false)
```
I'm afraid that someone would understand that he needs to add `--wide true`.
I think that how it is written the default value is implicit but understandable.

Reply:
@naveenpaul1 Would you please add your suggestion or you meant what I added?